bigchaindb/backend/localmongodb/query.py

```diff
@@ -266,6 +266,22 @@ def get_unspent_outputs(conn, *, query=None):
                                                   projection={'_id': False}))
 
 
+@register_query(LocalMongoDBConnection)
+def put_pre_commit_state(conn, state):
```

Comment:
All functions to save data have "store" as prefix, would it make sense to use it here as well so we are consistent?